kieserver/63/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerTestBase.java

```diff
@@ -131,29 +130,25 @@ public BatchExecutionCommand batchCommand() {
     }
 
     /*
-    * Verifies the KieContainer ID, it should be decisionserver-hellorules
+    * Verifies the KieContainer ID, it should be 41895328c0d0a1747e2b24d53cbbfeb2 for decisionserver-hellorules
+    *                                        and 5fe16e3ab6fc8ae7fbf9fd71b8eb57f1 for AnotherContainer
     * Verifies the KieContainer Status, it should be org.kie.server.api.model.KieContainerStatus.STARTED
     */
     public void checkDecisionServerContainer() throws Exception {
         log.info("Running test checkDecisionServerContainer");
+
         // for untrusted connections
         prepareClientInvocation();
 
-        List<KieContainerResource> kieContainers = getKieRestServiceClient(getRouteURL()).listContainers().getResult().getContainers();
-
-        // Sorting kieContainerList
-        Collections.sort(kieContainers, ALPHABETICAL_ORDER);
-        // verify the KieContainer Name
-
-        //When the multicontainer test is running, the decisionserver-hellorules will not be the first anymore
-        int pos = 0;
-        if (kieContainers.size() > 1) {
-            pos = 1;
-        }
-
-        Assert.assertTrue(kieContainers.get(pos).getContainerId().startsWith("decisionserver-hellorules_"));
-        // verify the KieContainer Status
-        Assert.assertEquals(org.kie.server.api.model.KieContainerStatus.STARTED, kieContainers.get(pos).getStatus());
+        getKieRestServiceClient(getRouteURL()).listContainers().getResult().getContainers().stream().forEach(container -> {
+            String containerAliasToID = new SumCoder.MD5().encode(container.getResolvedReleaseId().getArtifactId() + "=" + container.getReleaseId());
```

Comment:
I think we should not use exclusive features from Java 8, like lambdas. Could you refactor this statement to not use lambdas?